main/FatesInventoryInitMod.F90

```diff
@@ -1112,8 +1112,10 @@ subroutine set_inventory_edcohort_type1(csite,bc_in,css_file_unit,npatches, &
 
          call prt_obj%CheckInitialConditions()
 
+
          ! Since spread is a canopy level calculation, we need to provide an initial guess here.
-         call create_cohort(csite, cpatch, temp_cohort%pft, temp_cohort%n, temp_cohort%hite, temp_cohort%dbh, &
+call create_cohort(csite, cpatch, temp_cohort%pft, temp_cohort%n, temp_cohort%hite, &
```

Comment:
sorry, being very picky, could you fix indentation here

Reply:
Yikes, sorry. I'll fix that.